app/services/process_parser.rb

```diff
@@ -3,27 +3,65 @@
 ##
 # Parsing Process creation request
 class ProcessParser
-  attr_reader :process
+  attr_reader :process_element
+
+  PROCESS_ATTRIBUTES = %i[
```

Comment:
I believe this is the full list of attributes from the Java-based workflow service.